tests/integration/REST_API/Controllers/SettingsTest.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+use SkyVerge\WooCommerce\PluginFramework\v5_6_1\REST_API\Controllers\Settings;
+use SkyVerge\WooCommerce\PluginFramework\v5_6_1\Settings_API\Abstract_Settings;
+use SkyVerge\WooCommerce\PluginFramework\v5_6_1\Settings_API\Setting;
+use SkyVerge\WooCommerce\PluginFramework\v5_6_1\Settings_API\Control;
+
+/**
+ * Tests for the Abstract_Settings class.
+ *
+ * @see \SkyVerge\WooCommerce\PluginFramework\v5_6_1\REST_API\Controllers\Settings
+ */
+class SettingsTest extends \Codeception\TestCase\WPTestCase {
+
+
+	/** @var \IntegrationTester */
+	protected $tester;
+
+
+	/** @var Abstract_Settings */
+	protected $settings;
+
+
+	protected function _before() {
+
+		require_once 'woocommerce/rest-api/Controllers/Settings.php';
+		require_once 'woocommerce/Settings_API/Abstract_Settings.php';
+		require_once 'woocommerce/Settings_API/Control.php';
+		require_once 'woocommerce/Settings_API/Setting.php';
+	}
+
+
+	protected function _after() {
+
+	}
+
+
+	/** Tests *********************************************************************************************************/
+
+
+	/** @see Abstract_Settings::get_id() */
```

Comment:
Is this correct?

Reply:
Noup. I left lots of copypasta on this class :(. Fixed in a717049